perses/app/relative_point_mutation_setup.py

```diff
@@ -52,7 +52,7 @@ class PointMutationExecutor(object):
                                      ligand_index=0,
                                      forcefield_files=['amber14/protein.ff14SB.xml', 'amber14/tip3p.xml'],
                                      barostat=openmm.MonteCarloBarostat(1.0 * unit.atmosphere, temperature, 50),
-                                     forcefield_kwargs={'removeCMMotion': False, 'ewaldErrorTolerance': 1e-4, 'nonbondedMethod': app.PME, 'constraints' : app.HBonds, 'hydrogenMass' : 4 * unit.amus},
+                                     forcefield_kwargs={'removeCMMotion': False, 'ewaldErrorTolerance': 1e-4, 'nonbondedMethod': app.PME, 'constraints' : app.HBonds, 'hydrogenMass' : 3 * unit.amus},
```

Comment:
@zhang-ivy : Do `forcefield_kwargs` get dispatched to `openmm.app.ForceField` or to `openmmtools.generators.SystemGenerator`? If `SystemGenerator`, then `ewaldErrorTolerance` and `nonbondedMethod` should be in `periodic_forcefield_kwargs`. Worth double-checking to make sure we don't have issues with this elsewhere.

Reply:
They are passed into `SystemGenerator`. This docstring is not up to date. `nonbondedMethod` has already been moved to `periodic_forcefield_kwargs`, but I'll need to move `ewaldErrorTolerance` as well. Thanks!